packages/wdio-static-server-service/tests/__mocks__/express.ts

```diff
@@ -1,6 +1,12 @@
 const expressMock = jest.fn().mockReturnValue({
     use: jest.fn(),
-    listen: jest.fn().mockImplementation((port, cb) => {
+    listen: jest.fn().mockImplementation(function (this: unknown, port, cb) {
+        if (this === undefined) {
+            return cb(
+                new Error('This value missing when invoking server.listen()')
```

Comment:
There is no way to assert against the `this` value used when invoking `jest.fn()`, I added a check directly to mock to add future regression to avoid having to rewrite the entire mock.

IMO this is not the best approach. I am fine removing this check if needed.